tests/utils/test_databricks_utils.py

```diff
@@ -7,3 +11,61 @@ def test_no_throw():
     None.
     """
     assert not databricks_utils.is_in_databricks_notebook()
+
+
+@mock.patch('databricks_cli.configure.provider.get_config')
```

Comment:
These tests were more-or-less just moved from `rest_utils`, but the HTTP parts have been removed.